release-notes/unreleased/2014-update-i18next.yml

```diff
@@ -0,0 +1,7 @@
+issue_key: 2014
+show_in_stores: false
+platforms:
+  - web
+  - android
+  - ios
+en: Updating dependencies
```

Comment:
Maybe we should discuss release notes again. I think a release note like this does not add any benefit as there is no user facing changes/nothing testable :see_no_evil:

Reply:
I'm okay with not adding one 🤷